website/content/docs/integrations/consul/acl.mdx

```diff
@@ -0,0 +1,446 @@
+---
+layout: docs
+page_title: Consul ACL
+description: Learn how to integrate Nomad with Consul when ACL is enabled.
+---
+
+# Consul ACL
+
+The [Consul ACL][consul_acls] system protects the cluster from unauthorized
+access. When enabled, both Consul and Nomad must be properly configured in
+order for their integrations to work.
+
+Nomad agents must be configured with their own Consul ACL token, and Consul
+must be configured to accept workload identities from tasks and services
+running in Nomad.
+
+## Nomad Agents
+
+Nomad agents need access to Consul in order to register themselves in the
+service catalog and discover other Nomad agents via service discovery for
+automatic clustering. Nomad servers also create [configuration
+entries][consul_config_entry] for Consul Service Mesh, so the specific
+permissions vary slightly between Nomad servers and clients.
```

Comment:
Update to this. I wanted to avoid giving too much instructions here and present this more as a documentation of what Nomad needs to work. But happy to reword it the suggested way as well.

<img width="749" alt="image" src="https://github.com/hashicorp/nomad/assets/775380/09d6eecc-6c0f-4965-981f-5251abb24843">

Reply:
Perfect!